modules/builds-build-unprivileged.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * builds/advanced-build-operations.adoc
+
+:_content-type: PROCEDURE
+[id="builds-build-unprivileged_{context}"]
+= Unprivileged Builds (Developer Preview)
+
+Builds can be configured to run as an unprivileged user, with the minimal set of capabilities to build a container image.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+Unprivileged {product-title} Builds is a Developer Preview feature, intended to provide early access to new capabilities. This feature is not recommended for production or business-critical environments.
+====
+
+.Procedure
+
+* To run builds as an unprivileged user, pass the `BUILD_PRIVILEGED` environment variable as part of the `sourceStrategy`
+ifndef::openshift-online[]
+or `dockerStrategy`
+endif::[]
+in a `BuildConfig`, with the value `"false"`:
```

Comment:
or `Build`, unless that's assumed?

Reply:
`Build` is typically assumed. OpenShift CI may be one of the few users that doesn't use `BuildConfig` objects.